### PR TITLE
Allow symfony 4

### PR DESCRIPTION
**Pull request by caciobanu**

### CODE REVIEW

**Comment by caciobanu** [comment]
Build should pass after https://github.com/SimpleBus/AsynchronousBundle/pull/20 is merged and released.

**Comment by caciobanu** [comment]
@cmodijk any news on this also ?

File: composer.json [hunk 1]
```diff
@@ -35,5 +35,7 @@
         "psr-4": {
             "SimpleBus\\JMSSerializerBundleBridge\\Tests\\": "tests"
         }
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```
Comment (ruudk) [inline composer.json]: Since SF 4.0 is released, can we drop this?

**Comment by cmodijk** [comment]
@caciobanu The tests for this are failing that maybe have something to do with the fact that we need to tag other items first

File: composer.json (hunk 1 above)
Comment (caciobanu) [reply composer.json]: We can, but I think we can keep it to easily test with version of dependencies not yet released to prepare necessary changes for forward compatibility.

**Comment by caciobanu** [comment]
Yes, I think we have to trigger the build again after the other repo's get a new tag(release).

**Comment by cmodijk** [comment]
@ruudk Can we tag the missing items so we can merge this?

File: composer.json (hunk 1 above)
Comment (cmodijk) [reply composer.json]: I just removed these lines in https://github.com/SimpleBus/SymfonyBridge/pull/62 as these seems to create a memory problem in composer in php 5.6

**Comment by ruudk** [comment]
Tagged https://github.com/SimpleBus/AsynchronousBundle/releases/tag/v3.0.3
Restarted tests for this PR, now ✅

**Comment by cmodijk** [comment]
@ruudk Maybe we should still remove the `minimum-stability` and `prefer-stable` items?

**Comment by caciobanu** [comment]
If you guys want, I can make PR's for all repos and remove minimum-stability and prefer-stable.

**Comment by ruudk** [comment]
@caciobanu I want ahead and changed your PR to include the fixes (also different php version). Lets  wait for Travis to go Green so we can merge it :v:

**Comment by caciobanu** [comment]
OK. I will make the PRs for the other repos as soon as I will have some spare time.

**Comment by cmodijk** [comment]
@caciobanu If you want to do that sure!

**Comment by caciobanu** [comment]
After this is merged maybe you have the time to take a look at https://github.com/SimpleBus/SymfonyBridge/issues/64. I think that what I've written there are the next step for the Symfony 4 support to be complete.

**Comment by ruudk** [comment]
Merged in https://github.com/SimpleBus/JMSSerializerBundleBridge/pull/15.

Thanks!

**Pull request closed without merging**